src/quo2/components/dividers/date.cljs

```diff
@@ -0,0 +1,20 @@
+(ns quo2.components.dividers.date
+  (:require [react-native.core :as rn]
+            [clojure.string :as string]
+            [quo2.components.markdown.text :as quo2.text]
+            [quo2.foundations.colors :as quo2.colors]))
+
+(defn date [value]
+  [rn/touchable-without-feedback
+   {:on-press #(rn/dismiss-keyboard!)}
```

Comment:
not sure if this should be here or just part of the screen implementation. @OmarBasem, @ibrkhalil / anyone else working on messages any thoughts on this?

Reply:
yes we should pass on-press as a parameter, or even better to move touchable outside the component

Reply:
Moved touchable outside